docs/UserGuide.md

```diff
@@ -133,10 +133,11 @@ ___
 
 
 ### Contact Parameters
+
 | Parameter             | Description                                                                                                                                                                                                       | Examples                                                       |
 |-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------|
-| `NAME`                | The name of the contact. <br/> <br/> It should only contain alphanumeric characters and spaces, and it should not be blank.                                                                                       | <ul><li>`John Doe`</li><li>`Miguel-James Lee`</li></ul>        |
-| `PHONE`               | The phone number of the contact. <br/> <br/> It must contain only numbers, and be at least 3 digits long.                                                                                                         | <ul><li>`999`</li><li>`91243253`</li></ul>                     |
```

Comment:
Seeing if spaces are somehow needed for <ul> and <li> to render on product website. Worst case scenario mimic the behaviour using `<br> •`